test/unit/spec/MfaVerify_spec.js

```diff
@@ -3421,6 +3421,7 @@ function (Okta,
           });
 
           // Do this when we have implemented push errors in OktaAuth and have an example
+          // eslint-disable-next-line jasmine/no-disabled-tests
```

Comment:
We can probably get rid of this disabled test. Looks like this was added 4 years ago, along with the comment? https://github.com/okta/okta-signin-widget/blame/7074127a62c87f8e59030d26ef5c732594074c9d/test/unit/spec/MfaVerify_spec.js#L3423